bin/fusor-update-repos

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import os
+import json
+
+USER = 'admin'
+PASSWORD = 'changeme'
```

Comment:
Hardcoding password looks like a potential problem.

We should ask the user for credentials and pass those credentials to the script.